### ISSUE TITLE
std::bad_alloc

### ISSUE THREAD

**Issue by horizonheart** [arashno/tensorflow_multigpu_imagenet#13]
2018-08-15 12:23:05.209896: epoch 21, step 30, loss = 0.51, Top-1 = 0.78 Top-5 = 1.00 (46.9 examples/sec; 0.682 sec/batch)
2018-08-15 12:23:32.467195: epoch 21, step 40, loss = 0.29, Top-1 = 0.88 Top-5 = 1.00 (46.2 examples/sec; 0.693 sec/batch)
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc

**Comment by arashno** [comment]
I have never run into such a problem.
Could you please provide more information on this error? I mean the stack trace.
